src/core/store/reducers/globals.js

```diff
@@ -27,6 +29,10 @@ export default function globals(state = initialState, action) {
       return {...state, mobileConnected: action.payload}
     case constants.ACCEPT_TERMS:
       return {...state, termsAccepted: true}
+    case constants.PERMISSIONS_ASKED:
```

Comment:
is this something safe to save on redux? Can't a permission be reasked on some browsers (even on the same session)?